homepage.php

```diff
@@ -32,8 +32,8 @@
 				<a href = "homepage.php"><img src = "images/logo8.png" height = "100"></a>
 			</div>
 			<?php
-				//Pulls the links from the nav_normal.php page and places them in the navigation div
-				require 'nav_normal.php'; //'require' is 100% needed for the site to run 
+				//Pulls the links from the nav_admin.php page and places them in the navigation div
```

Comment:
2/9/2021: I change to link of the nav normal to nav admin because nav admin and allowed to access to other function which was delete user for example.